### PR TITLE
[Doc] Add two methods (log topic & CLI) for debugging functions

### PR DESCRIPTION
**Pull request by Anonymitaet**
@sijie could you please help review? Thanks

### CODE REVIEW

**Comment by sijie** [comment]
run java8 tests

File: site2/docs/functions-debugging.md [hunk 1]
```diff
@@ -124,3 +130,322 @@ To use localrun like above programmatically please addd the following dependency
...
+```
+
+As shown below, the `trigger` command returns the following result:
+
+```text
+This is my function!
+```
+
+> #### **Note**
+> You must specify the [entire topic name](getting-started-pulsar.md#topic-names) when using the `--topic` option. Otherwise, the following error will be raised.
```
Comment (Jennifer88huang) [inline site2/docs/functions-debugging.md]: will be --> is?

**Comment by Anonymitaet** [comment]
@Jennifer88huang Thanks for your review, all your comments have been incorporated.

File: site2/docs/functions-debugging.md [hunk 2]
```diff
@@ -121,6 +127,325 @@ To use localrun like above programmatically please addd the following dependency
...
+  "processingGuarantees": "ATLEAST_ONCE",
+  "retainOrdering": false,
+  "userConfig": {},
+  "runtime": "JAVA",
+  "autoAck": true,
+  "parallelism": 1
+}
+```
+
+### status
```
Comment (Jennifer88huang) [inline site2/docs/functions-debugging.md]: Status or status?

Comment (Jennifer88huang) [reply site2/docs/functions-debugging.md]: I see some similar cases for those titles, should we use lowercase or capitalize the first letter in the heading?

File: site2/docs/functions-debugging.md [hunk 3]
```diff
@@ -121,6 +127,325 @@ To use localrun like above programmatically please addd the following dependency
...
 ```
 
-For complete code samples, see [here](https://github.com/jerrypeng/pulsar-functions-demos/tree/master/debugging)
+For complete code samples, see [here](https://github.com/jerrypeng/pulsar-functions-demos/tree/master/debugging).
 
 In the future, debugging with localrun mode for Pulsar Functions written in other languages will be supported.
+
+## Use log topic
+
+Pulsar Functions allows you to output the log information defined in functions to a specified log topic. Consumers can be configured to consume messages from a specified log topic to check the log information.
```
[the remark this answers is not in the document]
Comment (Anonymitaet) [reply site2/docs/functions-debugging.md]: Thanks, I've fixed.

File: site2/docs/functions-debugging.md (hunk 2 above)
Comment (Anonymitaet) [reply site2/docs/functions-debugging.md]: I think we should use lowercase to keep consistent throughout, see [Pulsar Admin CLI](https://pulsar.apache.org/docs/en/pulsar-admin/).

**Comment by Anonymitaet** [comment]
@sijie could you please help merge? Thanks

File: site2/docs/functions-debugging.md (hunk 2 above)
Comment (Jennifer88huang) [reply site2/docs/functions-debugging.md]: Would you add `` for them, to indicate they are commands?

Comment (Anonymitaet) [reply site2/docs/functions-debugging.md]: Good suggestion, I've fixed.

**Comment by Jennifer88huang** [comment]
run java8 tests

**Pull request merged**